lib/commands/restart.js

```diff
@@ -6,11 +6,12 @@ class RestartCommand extends Command {
         const instance = this.system.getInstance();
 
         if (!instance.running()) {
-            return Promise.reject(new Error('Ghost instance is not currently running.'));
+            const StartCommand = require('./start');
+            this.ui.log('Ghost instance is not running! Starting...', 'yellow');
+            return this.runCommand(StartCommand, {quiet: true});
```

Comment:
the quiet option should be removed here imo - we want the start command to output the "Ghost instance has been started" message after Ghost starts.